tests/unit/test__legacy_entity_pb.py

```diff
@@ -57,23 +57,23 @@ def test_TryMerge_mutable_key_app():
         d = _get_decoder(b"\x6a\x03\x6a\x01\x41")
         entity.TryMerge(d)
         assert entity.key().has_app()
-        assert entity.key().app().decode() == "A"
+        assert entity.key().app.decode() == "A"
```

Comment:
Ah ha, here's the other half of those API changes 😆

Reply:
Yep. the new model uses props, so the ported legacy code needs to be massaged a bit to be compatible.